src/actions/admin.test.js

```diff
@@ -226,6 +226,20 @@ describe('Actions.Admin', () => {
         }
     });
 
+    it('testSiteURL', async () => {
+        nock(Client4.getBaseRoute()).
+            post('/site_url/test').
+            reply(200, OK_RESPONSE);
+
+        await Actions.testSiteURL('http://lo.cal')(store.dispatch, store.getState);
+
+        const state = store.getState();
+        const request = state.requests.admin.testSiteURL;
+        if (request.status === RequestStatus.FAILURE) {
```

Comment:
Here I would’ve check the result instead of the request status, as there is no need to track the request